src/collection.js

```diff
@@ -527,6 +527,15 @@ class Collection {
 
     return this
   }
+
+  /**
+   * Returns the average of all collection items
+   *
+   * @returns {Number}
+   * */
+  async avg () {
+    return this.sum() / this.size

```

Comment:
Typo, `this.size` is a method

```suggestion
    return this.sum() / this.size()
```